react-auth/src/index.tsx

```diff
@@ -1,26 +1,8 @@
 import React, { FC, createContext, useContext } from 'react';
 import { useAsync } from 'react-async-hook';
+import { Profile } from '@reshuffle/auth';
```

Comment:
I think `@reshuffle/react-auth` is quite generic, and it would work even if Profile looked like a generic `{}` object, and might be overridden by users - then we wouldn't need to export a package...

Reply:
I'd rather give our users a consistent Profile interface.
In the future we might want to let users add more information to the basic profile.